AccountService/src/UserInputService.java

```diff
@@ -68,10 +76,57 @@ public void mainUserInputFlow() {
 		System.out.println("Have you got an account? (YES/NO) ");
 		String isAccountExist = userInput.nextLine();
 
+		User currentUser;
 		if (isAccountExist.equals("YES")) {
-			loginInput();
+			currentUser = loginInput();
 		} else {
-			createAccountInput();
+			currentUser = createAccountInput();
+		}
+		System.out.println("What's action do you want to do now");
+		if(currentUser.getAdmin() == true){
+			System.out.println("1. Print all customer's information");
+			System.out.println("2. Delete customer's information");
+			System.out.println("3. Print selected customer details");
+			System.out.println("Please choose a number for the action you want");
+			String adminSelectOption;
+			adminSelectOption = userInput.nextLine();
+			if(adminSelectOption.equals("1")){
+				customerService.printAllCustomersInfo();
+			}
+			if(adminSelectOption.equals("2")){
+				System.out.println("Please choose customer you want to delete");
+				System.out.println("Customer name: ");
+				String deleteCustomer;
+				deleteCustomer = userInput.nextLine();
+				if(loginService.verifyUsername(deleteCustomer)){
+					loginService.deleteSelectCustomerInfo(deleteCustomer);
+				}
+				else {
+					System.out.println("Account name does not exist");
+				}
+			}
+			if(adminSelectOption.equals("3")){
+				System.out.println("PLease choose customer you want to print out");
+				String printCustomer;
+				printCustomer = userInput.nextLine();
+				if(loginService.verifyUsername(printCustomer)){
+						loginService.printSelectCustomerInfo(printCustomer);
+				}
+			}
 		}
+		else{
+			System.out.println("1. Print your information");
+			System.out.println("2. Delete your information");
+			System.out.println("Please choose a number for the action you want");
+			String customerSelectOption;
+			customerSelectOption = userInput.nextLine();
+			if(customerSelectOption.equals("1")){
+				loginService.printSelectCustomerInfo(currentUser.getUserName());
+			}
+			if(customerSelectOption.equals("2")){
+				loginService.deleteSelectCustomerInfo(currentUser.getUserName());
+			}
```

Comment:
Use switch statement for adminSelectOption. This avoids repetition of adminSelectOption.equals()